src/lib/types/restrictions.ts

```diff
@@ -94,7 +92,6 @@ export const stepValidation = z
   .object({
```

Comment:
do we need additional entries here for the new restrictions?

Reply:
I'll add them

Reply:
they're added